api_blog/validators.py

```diff
@@ -0,0 +1,20 @@
+from marshmallow import Schema, ValidationError, fields, validates_schema
+
+
+class UserValidation(Schema):
+    username = fields.String()
+    password = fields.String()
+
+    @validates_schema
+    def valid_fields(self, data, **kwargs):
+        if len(data) != 2:
+            raise ValidationError("you mast fill two field: username and password")
+
+        name = data["username"]
+        password = data["password"]
+
+        if not 3 <= len(name) <= 50:
+            raise ValidationError("username: min length 3, max length 50")
+
+        if not 8 <= len(password) <= 25:
+            raise ValidationError("password: min length 8, max length 25")
```

Comment:
Remove this method at all. Use String(required=True) to check that the field is in the data. Use marshmallow validators to check the length. Here is about [marshmallow validators](https://marshmallow.readthedocs.io/en/stable/marshmallow.validate.html)